### PR TITLE
Add xonsh-autoxsh xontrib

### PR DESCRIPTION
**Pull request by Granitas**
Hi,
I've made a clone of [autoenv](https://github.com/kennethreitz/autoenv) for xonsh. It functions very much the same. 
If anyone has any comments or suggestions - let me know!

### CODE REVIEW

**Comment by scopatz** [comment]
Thanks @Granitas!  It seems like you are missing a comma in the JSON here.  Also, you only seems to have the package name.  Would you also please add an entry for the xontrib, farther up in the file. See this for more detail: http://xon.sh/tutorial_xontrib.html#tell-us-about-your-xontrib

File: xonsh/xontribs.json [hunk 1]
```diff
@@ -83,6 +83,12 @@
    "install": {
     "pip": "pip install xonsh-pacman-tabcomplete"
    }
```
Comment (scopatz) [inline xonsh/xontribs.json]: comma needed here.

**Comment by Granitas** [comment]
@scopatz Sorry, json with it's double quotes and irregular commas, is the bane of my existence. Though it seems like I won the battle this time around and holy inspector Travis has deemed my contribution worthy.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**